addon/types/map_export/brush.py

```diff
@@ -52,7 +66,7 @@ def calc_uv_axes(settings: typing.Any, bm: bmesh.types.BMesh, face: bmesh.types.
         for loop in face.loops[0:3]:
             uv = loop[uv_layer].uv
             u_vals.append(uv[0])
-            v_vals.append(uv[1])
+            v_vals.append(1 - uv[1])
```

Comment:
Would this have been necessary with my old code too, or is it required because of something in your math?
Just curious; doesn't really matter.